modular_skyrat/modules/cargo/code/goodies.dm

```diff
@@ -111,3 +130,13 @@
 	name = "Blue Carpet Single-Pack"
 	contains = list(/obj/item/stack/tile/carpet/blue/fifty)
 
+/*
+*	MEDICAL STUFF
+*/
+
+
+/datum/supply_pack/goody/afad
+	name = "Automated First Aid Device"
+	desc = "Someone hurt, and it's to much of a bother to manually handle their burns, or cuts? Look no further then the AFAD, a state of the art first-aid device!"
```

Comment:
```suggestion
	desc = "Someone mildly hurt, and it's to much of a bother to manually handle their burns or cuts? Look no further then the AFAD, a state of the art pain-relief device!"
```
First-Aid implies its used for, well, first aid - treating life-threatening wounds. Considering its nerfed to only relieve minor damage, I dont think First-Aid applies.